### PR TITLE
Update About Us

### LINKED ISSUE #11: Update about us
Each person to add their about us and readme.

- [x] Drustan
- [x] Jia Hui
- [x] Charmaine
- [x] Jayne

Find in docs/AboutUs.md

### PR DESCRIPTION
**Pull request by jiahui0309**
Closes #11

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -9,51 +9,39 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
```
Comment (drustanyjt) [inline docs/AboutUs.md:9]: Can change/remove email since it's not our email,

File: docs/AboutUs.md [hunk 2]
```diff
@@ -9,51 +9,39 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-[[portfolio](team/johndoe.md)]
+[[github](http://github.com/jiahui0309)] 
+[[linkedin](https://www.linkedin.com/in/song-jia-hui/)]
 
 * Role: Developer
-* Responsibilities: Dev Ops + Threading
+* Responsibilities: Integration
 
-### James Doe
+### Charmaine Teng Shi Rou
```
Comment (tengcharmaine) [inline docs/AboutUs.md]: Can change the order of my name to Teng Shi Rou Charmaine! And on second thought can you also add my linked in (this is the link: www.linkedin.com/in/charmaine-teng) Thanks :)

File: docs/AboutUs.md (hunk 1 above)
Comment (drustanyjt) [reply docs/AboutUs.md:9]: Maybe put one of ours instead of `seer[at]comp.nus.edu.sg`?

File: docs/AboutUs.md [hunk 3]
```diff
@@ -9,51 +9,39 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Drustan Yeo
```
Comment (tengcharmaine) [inline docs/AboutUs.md]: I think change drustan's name to his full name also (Yeo Jun Teng Drustan)

File: docs/AboutUs.md [hunk 4]
```diff
@@ -9,51 +9,39 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
-[[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/drustanyjt)]
+[[linkedin](https://www.linkedin.com/in/drustan/)]
 
 * Role: Team Lead
-* Responsibilities: UI
 
-### Johnny Doe
+### Jayne Chen
```
Comment (tengcharmaine) [inline docs/AboutUs.md]: Jayne's also (Chen Jiaying Jayne)

**Review by drustanyjt — CHANGES_REQUESTED** [review]
Changes look good, but do we need to remove the `seer` email?

**Review by tengcharmaine — COMMENTED** [review]
Can change to full name for all and on second thought help me add my linked in also please :)

**Review by drustanyjt — APPROVED** [review]
LGTM!

**Comment by drustanyjt** [comment]
Closes #11.

**Pull request merged**